### PR TITLE
[DOCFIX] Update HDFS doc

### PR DESCRIPTION
**Pull request by dbw9580**
Minor updates to HDFS doc.

### CODE REVIEW

File: docs/en/ufs/HDFS.md [hunk 1]
```diff
@@ -144,15 +143,16 @@ set the property `alluxio.master.mount.table.root.option.alluxio.underfs.hdfs.co
 Make sure this configuration is set on all servers running Alluxio.
 
 ```
-alluxio.underfs.hdfs.configuration=/path/to/hdfs/conf/core-site.xml:/path/to/hdfs/conf/hdfs-site.xml
+alluxio.master.mount.table.root.option.alluxio.underfs.hdfs.configuration=/path/to/hdfs/conf/core-site.xml:/path/to/hdfs/conf/hdfs-site.xml
```
Comment (dbw9580) [inline docs/en/ufs/HDFS.md:146]: This one escaped in https://github.com/Alluxio/alluxio/pull/13139

File: docs/en/ufs/HDFS.md [hunk 2]
```diff
@@ -168,11 +168,11 @@ HDFS is consistent with Alluxio (e.g., a file created by user Foo in Alluxio is
 HDFS also with owner as user Foo), the user to start Alluxio master and worker processes
 **is required** to be either:
 
-1. [HDFS super user](http://hadoop.apache.org/docs/r2.7.2/hadoop-project-dist/hadoop-hdfs/HdfsPermissionsGuide.html#The_Super-User).
+1. [HDFS super user](http://hadoop.apache.org/docs/r3.3.1/hadoop-project-dist/hadoop-hdfs/HdfsPermissionsGuide.html#The_Super-User).
```
Comment (dbw9580) [inline docs/en/ufs/HDFS.md:171]: Pending https://github.com/Alluxio/alluxio/pull/15395

File: docs/en/ufs/HDFS.md [hunk 3]
```diff
@@ -232,7 +232,9 @@ $ mvn -T 4C clean install -Dmaven.javadoc.skip=true -DskipTests \
 ```
 
 #### Using Mount Command-line
-When using the mount Alluxio shell command, one can pass through the mount option `alluxio.underfs.version` to specify which version of HDFS to mount. If no such a version is specified, by default Alluxio treats it as Apache HDFS 2.7.
+When using the mount Alluxio shell command, one can pass through the mount option `alluxio.
+underfs.version` to specify which version of HDFS to mount. If no such a version is specified, 
+by default Alluxio treats it as Apache HDFS 3.3.
```
Comment (dbw9580) [inline docs/en/ufs/HDFS.md:237]: Update default HDFS version to 3.3

**Review by Xenorith — APPROVED** [review]

**Comment by Xenorith** [comment]
alluxio-bot, merge this please

**Comment by Xenorith** [comment]
alluxio-bot, cherry-pick this to branch-2.8 please

**Pull request merged**